### PR TITLE
added calendar events resource

### PR DESCRIPTION
**Pull request by 6reduk**

### CODE REVIEW

File: src/Factory.php [hunk 1]
```diff
@@ -84,10 +85,10 @@ public static function createWithToken($token, $client = null, $clientOptions =
      * @param  array   $arguments
      * @return \SevenShores\Hubspot\Resources\Resource
      */
-    function __call($name, $arguments = null)
+    public function __call($name, $arguments = null)
```
Comment (ryanwinchester) [inline src/Factory.php]: hey, we don't need that 😂

**Comment by ryanwinchester** [comment]
@6reduk please remove the public keyword, i left it out on purpose.

**Comment by 6reduk** [comment]
Hello, Ryan.
I remove redundant keyword.

**Comment by ryanwinchester** [comment]
thanks :)

**Pull request merged**